### ISSUE TITLE
Test change in contents referenced by aria-describedby 

### ISSUE THREAD

**Issue by mfairchild365** [accessibilitysupported/a11ysupport.io#181]
Most screen readers announce changes. This behavior should be tested and documented across screen readers.
